frontend/src/SnippetManager.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { ImageProvider } from "./ImageProvider";
+import { SocialMediaAccount } from "./SocialMediaAccountsListTile";
+
+export interface Param {
+    name: string
+    value: string
+}
+
+/**Contains a bunch of useful snippets to be standardized across the application. */
+export class SnippetManager {
+
+    /**Create an info message with a documentation link and a tooltip. */
+    static createInfo(url: string, text: string, cssClasses?: string, tooltip?: string, cssClassesToolTipText?: string) {
+        const className = "tooltip " + cssClasses;
+        const classNameText = "tooltiptext " + cssClassesToolTipText;
+        return <span className={className}>{text}
+            {text && <span><a target="blank" href={url}>here</a>.&nbsp;</span>}
+            <a target="blank" href={url}>{ImageProvider.getImage("info-text")}</a>
+            <span className={classNameText}>{tooltip ? tooltip : text}</span>
+        </span>
```

Comment:
Children should better be in a new line.
```suggestion
        return <span className={className}>
            {text}
            {text && <span><a target="blank" href={url}>here</a>.&nbsp;</span>}
            <a target="blank" href={url}>{ImageProvider.getImage("info-text")}</a>
            <span className={classNameText}>{tooltip ? tooltip : text}</span>
        </span>
```